### DISCUSSION TITLE
How to make RSS feed to NOT include draft pages?

### DISCUSSION THREAD

**Discussion by hellt** [squidfunk/mkdocs-material#4490] [Q&A] [answered]
Hi all,
Does anyone know how to make sure that draft pages won't be included in the RSS feed?

**Comment by squidfunk** [comment]
I think it's best to ask the maintainers of the plugin, i.e., how to exclude pages via `meta`. Other than that, you might just move your drafts into a separate folder and [exclude the folder](https://guts.github.io/mkdocs-rss-plugin/configuration/#filter-pages). Please share your solution when you reach one.

**Comment by hellt** [comment, marked as answer]
added in mkdocs-rss plugin in version 1.5.0 today
